README.rst

```diff
@@ -101,7 +101,41 @@ manner.
 
 - `workflow.yaml <workflow/yadage/workflow.yaml>`_
 
-**FIXME**
+Since yadage only accepts one input directory as parameter we are going to create a wrapper directory which will contain links to `inputs` and `outputs` directories:
+
+.. code-block:: console
+
+   $ mkdir -p yadage-inputs/code yadage-inputs/inputs
+   $ ln code/helloworld.py  yadage-inputs/code/helloworld.py
+   $ ln inputs/names.txt  yadage-inputs/inputs/names.txt
+
+Once we have done this, we can run yadage providing this convenience directory as input.
+
+.. code-block:: console
+
+    $ yadage-run . workflow/yadage/workflow.yaml -p sleeptime=2 -p inputfile=inputs/names.txt -p helloworld=code/helloworld.py -d initdir=`pwd`/yadage-inputs
+    2018-01-26 09:07:16,816 - yadage.steering_object - INFO - initializing workflow with {u'inputfile': '/Users/rodrigddi/reana/reana-demo-helloworld/yadage-inputs/inputs/names.txt', u'helloworld': '/Users/rodrigdi/reana/reana-demo-helloworld/yadage-inputs/code/helloworld.py', u'sleeptime': 2}
+    2018-01-26 09:07:16,816 - yadage.steering_api - INFO - running yadage workflow workflow/yadage/workflow.yaml on backend <yadage.backends.packtivitybackend.PacktivityBackend object at 0x108d02f10>
+    2018-01-26 09:07:16,817 - adage.pollingexec - INFO - preparing adage coroutine.
+    2018-01-26 09:07:16,817 - adage - INFO - starting state loop.
+    2018-01-26 09:07:17,052 - adage.node - INFO - node ready <YadageNode init SUCCESS lifetime: 0:00:00.190794  runtime: 0:00:00.024573 (id: 6581a2ac163e16dcba29b4d92e417ffed315e219) has result: True>
+    2018-01-26 09:07:17,100 - packtivity_logger_helloworld.step - INFO - prepare pull
+    2018-01-26 09:07:19,139 - packtivity_logger_helloworld.step - INFO - running job
+    2018-01-26 09:08:24,782 - adage.node - INFO - node ready <YadageNode helloworld SUCCESS lifetime: 0:01:07.755887  runtime: 0:01:07.729937 (id: 14dc4c62c30dbabbe904377258c4adab7f40fc1b) has result: True>
+    2018-01-26 09:08:24,783 - adage.controllerutils - INFO - no nodes can be run anymore and no rules are applicable
+    2018-01-26 09:08:24,783 - adage.pollingexec - INFO - exiting main polling coroutine
+    2018-01-26 09:08:24,783 - adage - INFO - adage state loop done.
+    2018-01-26 09:08:24,784 - adage - INFO - execution valid. (in terms of execution order)
+    2018-01-26 09:08:24,784 - adage.controllerutils - INFO - no nodes can be run anymore and no rules are applicable
+    2018-01-26 09:08:24,784 - adage - INFO - workflow completed successfully
+
+We can now check if the output corresponds to the expected value. Since `yadage` can not be configured yet to write output to a specific directory, we should look for the directory with the name we have given to the workflow inside analysis root, `helloworld` in this case:
+
+.. code-block:: console
+
+   $ cat helloworld/greetings.txt
```

Comment:
It looks a bit weird to have `helloworld` directory, but calling it outputs wouldn't be nice either...

What about wrapping this all into a workdir, so that the cwd won't be polluted? Something like:

```console
$ mkdir -p yadage-local-run/yadage-inputs
$ cd yadage-local-run
$ cp -a ../code ../inputs yadage-inputs
$ yadage-run . ../workflow/yadage/workflow.yaml -p sleeptime=2 -p inputfile=inputs/names.txt -p helloworld=code/helloworld.py -d initdir=`pwd`/yadage-inputs
```

This works for me locally.  Could call it `yadage-local-run`, `test-yadage`, or something...

Reply:
Great, that's way better without polluting the analysis directory with `yadage` metadata directories. I have also tested it locally and it works, I am changing it.